### PR TITLE
Made connection pool configurable.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/ConnectionPoolConfigurationProperties.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package io.eventuate.local.common;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.Collections;
+import java.util.Map;
+
+@ConfigurationProperties(prefix = "connection")
```
Comment (cer) [inline eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/ConnectionPoolConfigurationProperties.java]: eventuate.cdc.connection

File: eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/ConnectionPoolConfigurationProperties.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+public class ConnectionPoolConfigurationProperties {
+  private Map<String, String> pool = Collections.emptyMap();
+
+  public Map<String, String> getPool() {
```
Comment (cer) [inline eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/ConnectionPoolConfigurationProperties.java]: getProperties()

**Comment by cer** [comment]
There also need to be sensible defaults.
What's the hikari default min size?
Does the max size of 10 make any sense for the CDC?

**Comment by dartartem** [comment]
hikari default min size is 10. Probably we can make it single sized by default. In case if cdc needs more it will create a few more.

File: eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/ConnectionPoolConfigurationProperties.java (hunk 2 above)
Comment (dartartem) [reply eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/ConnectionPoolConfigurationProperties.java]: ok

File: eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/ConnectionPoolConfigurationProperties.java (hunk 1 above)
Comment (dartartem) [reply eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/ConnectionPoolConfigurationProperties.java]: ok

**Pull request merged**